test/functional/page_objects/dashboard_page.js

```diff
@@ -525,8 +525,10 @@ export function DashboardPageProvider({ getService, getPageObjects }) {
     async filterOnPieSlice(sliceValue) {
       log.debug(`Filtering on a pie slice with optional value ${sliceValue}`);
       if (sliceValue) {
+        log.debug('Clicking slice');
         await testSubjects.click(`pieSlice-${sliceValue}`);
       } else {
+        log.debug('Could not find slice');
```

Comment:
This isn't really that the slice couldn't be found but that a specific slice value to click wasn't supplied, so it's going to just pick one randomly.